code/src/index.css

```diff
@@ -5,9 +5,212 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3) ), url('./assets/sparc-bg.jpg');
```

Comment:
I don't understand why you have a gradient with two times the same color.

Reply:
Did try to remove one of them and my image disappears, didn't realise that before :\